src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IInterpolatedStringOperation.cs

```diff
@@ -4749,12 +4749,89 @@ public void InterpolationEscapeConstantValue_WithoutDefaultHandler()
           null
       IInterpolatedStringTextOperation (OperationKind.InterpolatedStringText, Type: null) (Syntax: ' }}')
         Text:
-          ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: "" }}"", IsImplicit) (Syntax: ' }}')
+          ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: "" }"", IsImplicit) (Syntax: ' }}')
 ";
 
             VerifyOperationTreeAndDiagnosticsForTest<InterpolatedStringExpressionSyntax>(new[] { code }, expectedOperationTree, expectedDiagnostics, parseOptions: TestOptions.RegularPreview);
         }
 
+        [Fact, WorkItem(54703, "https://github.com/dotnet/roslyn/issues/54703")]
+        public void RawInterpolationEscapeConstantValue_WithoutDefaultHandler1()
+        {
+            var code = @"
+int i = 1;
+System.Console.WriteLine(/*<bind>*/$$""""""{{i}}""""""/*</bind>*/);";
+
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            string expectedOperationTree = @"
+IInterpolatedStringOperation (OperationKind.InterpolatedString, Type: System.String) (Syntax: '$$""""""{{i}}""""""')
+  Parts(1):
+      IInterpolationOperation (OperationKind.Interpolation, Type: null) (Syntax: '{{i}}')
+        Expression:
+          ILocalReferenceOperation: i (OperationKind.LocalReference, Type: System.Int32) (Syntax: 'i')
+        Alignment:
+          null
+        FormatString:
+          null";
+
+            VerifyOperationTreeAndDiagnosticsForTest<InterpolatedStringExpressionSyntax>(new[] { code }, expectedOperationTree, expectedDiagnostics, parseOptions: TestOptions.RegularPreview);
```

Comment:
nit: `parseOptions: TestOptions.RegularPreview` should no longer be needed (we've switched `CreateCompilation` to use that by default)

Reply:
done.

Reply:
Did you push the commit?